exonum-java-binding-core/rust/ejb-app/src/main.rs

```diff
@@ -1,12 +1,36 @@
+extern crate exonum_btc_anchoring;
 extern crate exonum_configuration;
 extern crate java_bindings;
+extern crate serde;
+#[macro_use]
+extern crate serde_derive;
 
+use exonum_btc_anchoring::ServiceFactory as BtcAnchoringServiceFactory;
 use exonum_configuration::ServiceFactory as ConfigurationServiceFactory;
+use java_bindings::exonum::helpers::config::ConfigFile;
 use java_bindings::exonum::helpers::fabric;
 
+const PATH_TO_SERVICES_TO_ENABLE: &str = "ejb_app_services.toml";
```

Comment:
Is it expected to be in the current working directory?

Reply:
Yes